### DISCUSSION TITLE
Why all Stalker modders can't join forces and use UE5 to build "remake" at least?

### DISCUSSION THREAD

**Discussion by mrNo0b** [OGSR/OGSR-Engine#460] [General]
Why everyone insists on working with XRay engine and all that suffering when its possible to use Unreal Engine 5?
All the time that you have spent collectively on modding this old game could be used to create near AAA game with UE5.

I ask this question every time I hear about another Stalker mod so I decided to ask here, maybe I will finally get the answer...

**Comment by joye-ramone** [comment]
so go ahead and start work on UE5. why every time I heard just questions )

**Comment by mrNo0b** [comment]
I am not a modder/game dev, also I don't mod Stalker games, I would never do that...
I want to know why using UE5 is a problem for people/teams who mod Stalker games.

**Comment by joye-ramone** [comment]
there is no problem in general. people do what they like. and what they can
if no one mod on UE yet that just mean that no one interesting in it so far
if you question is more to OGSR team - answer pretty same - no one interesting or have enough knowledge and exp with ue5 at this point and not one really interesting in getting it

**Comment by xrSimpAI** [comment]
For example, I am a C++ developer. What should I do in UE5? UE5 is primarily a ready-made engine for game designers who can sit in the editor for hours and create games.
in OGSR, I can constantly do something in the engine, which is interesting to me.
And in general, I am skeptical about creating "remakes" of the stalker on other engines. So far, not a single such project has survived to release.

**Comment by mrNo0b** [comment]
UE5 is using C++ as programming language, its C++ but with UE specific macros etc. every game requires programming,
UE has blueprint visual scripting but its less "effective" than C++ code.
the level of gameplay complexity will determine how much coding will be required.
Also UE5 is open source so you can edit the engine code to add something specific for Stalker game.
btw UE5 is using one file per actor, it allows better collaboration.

To make project survive there should be well designed roadmap to follow, and team manager - someone who will manage the communication between team members and keep the development on track and will make the decision when its required.
Team members will vote and select who will be the manager (person with the right skills (preferably) and ability to manage teams).
Also team members will be required to have discipline and respect the authority of the manager and leave the ego games for other projects.

To speed up dev time its possible to use UE Marketplace assets, Quixel Megascans, Metahuman to create NPC characters etc.

**Comment by joye-ramone** [comment]
A lot of things should be. but nothing exists yet. we all wait for this great time

**Comment by ramseysparrow** [comment]
I like the 'at least' part of the original question :) 
As if rebuilding a complete game with all its minute mechanics, from scratch, in a completely different engine is the obvious thing to do over improving the existing material within the original engine.

The good news is that someone is actually doing exactly that. The team is called GSC and their project is called 'Stalker 2'.

**Reply by mrNo0b** [reply]
If we will sum the number of modders x hours they spent on it we will probably get number higher than the time it took the original GSC to build the first Stalker game.
And now we have access to leading game engine, open source software, AI, premade assets, etc.
Time has comes to upgrade.

Stalker 2 "dead" for me is , I don't consider it as Stalker game, it will have no soul and no atmosphere, only modders can make this game what it should be.
